src/mono/mono/mini/mini-wasm-debugger.c

```diff
@@ -64,11 +65,13 @@ extern void mono_wasm_add_properties_var (const char*, gint32);
 extern void mono_wasm_add_array_item (int);
 extern void mono_wasm_set_is_async_method (guint64);
 extern void mono_wasm_add_typed_value (const char *type, const char *str_value, double value);
+extern void mono_wasm_add_files (const char *assembly_data, guint32 assembly_len, const char *pdb_data, guint32 pdb_len);
```

Comment:
maybe `mono_wasm_assembly_loaded_event`

Reply:
Opted for `mono_wasm_asm_loaded` since longer method names cause issues with the function registration. @thaystg was able to help me resolve this earlier when the method name was `mono_wasm_add_lazy_loaded_files`.

Reply:
No no, the problem wasn't the function name, was that the function was inside MONO variable. You can use the original name if you want